Environment_Engine/Query/Space.cs

```diff
@@ -44,7 +44,14 @@ public static partial class Query
         [Output("panelsAsSpace", "A collection of Environment Panels which have the given space name as a connected space")]
         public static List<Panel> ToSpace(this List<Panel> panels, string spaceName)
         {
-            return panels.Where(x => x.ConnectedSpaces.Contains(spaceName)).ToList();
+            List<Panel> result= panels.Where(x => x.ConnectedSpaces.Contains(spaceName)).ToList();
+            if(result.Count==0)
+                BH.Engine.Reflection.Compute.RecordWarning("Could not find following space: " + spaceName);
+            if(result.Count > 0 && result.Count<5)
+                BH.Engine.Reflection.Compute.RecordWarning("This space have less than 5 planar panels " + result.Count +" : "+ spaceName);
```

Comment:
This error message needs fixing - we have never specified anything about `planar` panels previously, even if they should be planar, this is a random place to start using the word so remove that word. Just keep it at "less than 5 panels". Format the error message better to say what the `result.Count` refers to.

Reply:
done